controllers/etcd_controller_test.go

```diff
@@ -106,7 +105,7 @@ var _ = Describe("Druid", func() {
 			s := &appsv1.StatefulSet{}
 			getStatefulset(c, instance, s)
 			testLog.Info("fetched Statefulset", "statefulset", s.Name)
-			Eventually(len(s.OwnerReferences), timeout).ShouldNot(BeZero())
+			Eventually(checkForEtcdAnnotations(s.GetAnnotations(), instance), timeout).Should(BeTrue())
```

Comment:
Make `checkForEtcdAnnotations ` public in controllers package. And keep the package names here as `controllers_test`

Reply:
I made this change so that I could unit test private functions as well. Ill mark the private functions in the test files with a `fortest` suffix. WDYT?